Update troubleshooting docs (#2312)
sublimelsp/LSP: docs/src/troubleshooting.md

## Self-help instructions

> To get more visibility into the inner-workings of the LSP client and the server and be able to diagnose problems, run `LSP: Toggle Log Panel` from the Command Palette. It might be a good idea to restart Sublime Text and reproduce the issue again, so that the logs are clean.

Author: ```suggestion
To see the LSP server and client communication, run `LSP: Toggle Log Panel` from the Command Palette. Logs are useful to diagnose problems.

!!! note
    It might be a good idea to restart Sublime Text and reproduce the issue again so that the logs are clean.
```

I would actually rewrite it as :point_up_2:
